app/assets/javascripts/spree/backend/spree_volume_pricing.js

```diff
@@ -87,4 +87,34 @@ $(document).ready(function () {
       }
     });
   });
+
+  // Added custom function to remove volume price row when delete action is performed
+  $('.spree_remove_volume_price').click( function() {
+    var $el = $(this);
+    $el.prev("input[type=hidden]").val("1");
```

Comment:
Rather than put all of this logic here why don't we just render something on the button itself so it knows what to delete?  That would be much more reliable than navigating the dom.  Just put a data-href on the button or something since we control the rendering of the delete button.

Reply:
@brettallred  According to my understanding are you referring to server-side rendering to remove content from DOM when we click delete button? Actually, this is Spree inbuilt function and I just used it as it is and added

`this.nextElementSibling.remove();`

It was causing the issue when while updating as it was sending deleted item ID to update